pipeline-braille-scripts/epub3-to-pef/src/main/resources/css/default.scss

```diff
@@ -0,0 +1,97 @@
+/*
```

Comment:
The reason why it's a bit weird is not so much because the CSS and documentation are in different modules, but more because the parent script is documentation-only I think. If it's possible to share the documentation of an option it should also be possible to share the implementation. The parent script can not be actually invoked, but maybe if we would put the common CSS in xml-to-pef that would improve things.